.travis.yml

```diff
@@ -18,16 +18,17 @@ matrix:
       python: pypy
     - os: linux
       python: pypy3
-# Perform the manual steps on osx to install python3 and activate venv
 before_install:
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update          ; fi
+  # Perform the manual steps on osx to install python3 and activate venv
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install python3; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then virtualenv venv -p python3; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then source venv/bin/activate; fi
+  - if [[ "$TRAVIS_PYTHON_VERSION" == "pypy3" ]]; then sudo apt-get -qq update; fi
+  - if [[ "$TRAVIS_PYTHON_VERSION" == "pypy3" ]]; then sudo apt-get install -y pypy3-dev; fi
```

Comment:
pypy3 is set, and we can't do much here. Alternative would be to go for similar build like macosx. Although running builds on containers on travis makes us ditch sudo (2nd line)